### DISCUSSION TITLE
Are you good at swimming? (水泳は得意ですか?)

### DISCUSSION THREAD

**Discussion by rktymt** [NAIST-SE/AttractingContributors#474] [General]
I've been a swimmer for over 10 years, but I'm still not good at backstroke (by the way, my specialty is breaststroke).

Is there any swimming method that you are not good at?

↓ Please enter from here.
https://github.com/rktymt/AttractingContributors_Question

**Comment by aru1702** [comment]
Interesting question which opposite to the title which I don't know how answer it (just kidding)
I rare swimming, I can only do normal one (like breaststroke, crawlstroke), but not expert them

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407
